tracer/src/Datadog.Trace/ContinuousProfiler/ConfigurationKeys.cs

```diff
@@ -11,5 +11,6 @@ internal static class ConfigurationKeys
         public const string CodeHotspotsEnabled = "DD_PROFILING_CODEHOTSPOTS_ENABLED";
         public const string EndpointProfilingEnabled = "DD_PROFILING_ENDPOINT_COLLECTION_ENABLED";
         public const string SsiDeployed = "DD_INJECTION_ENABLED";
+        public const string SkippedMethods = "DD_PROFILER_SKIPPED_METHODS";
```

Comment:
Does this need to be added to the config telemetry in dd-go?

Reply:
Good point. It does, I'll add that 👍